request-subgraph-storage/README.md

```diff
@@ -1,14 +1,22 @@
-[![Docker](https://badgen.net/badge/icon/docker?icon=docker&label)](https://hub.docker.com/r/requestnetwork/request-subgraph-storage)
+# Request Subgraph Storage
 
-# Request Subgraph: Storage
+## Description
+This image provides the means to create & deploy a subgraph to a graph node via a container job.
 
-A Docker image to deploy Request's storage subgraph to a TheGraph node.
+## Configuration
+subgraph data
+```bash
+ENV SUBGRAPH_REPO=      # defaults to https://github.com/RequestNetwork/storage-subgraph.git
+ENV SUBGRAPH_BRANCH=    # defaults to main
+ENV SUBGRAPH_FILE=      # no default, typically subgraph-<chain>.yaml for above repo
+ENV SUBGRAPH_NAMESPACE= # defaults to request-network/request-storage
+```
+graph-node / ipfs
```

Comment:
Please add a blank line between a code block and the next section, for readability. (please ignore the `\`, added only so the suggestion would display nicely.)
```suggestion
\```

graph-node / ipfs
```